src/main/java/com/google/codeu/data/Guides.java

```diff
@@ -10,24 +10,29 @@
     private String gender;
     private String location;
     private double charge;
+    private double alat;
+    private double along;
     private long timestamp;
-
+    private static double defaultLat = 23.7474824;
+    private static double defaultLong = 90.36420079999999;
     /**
      * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
      * random ID and uses the current system time for the creation time.
      */
-    public Guides(String name, String address, double contact_no, String gender, String location, double charge ) {
-        this(UUID.randomUUID(), name, address, contact_no, gender, location, charge, System.currentTimeMillis());
+    public Guides(String name, String address, double contact_no, String gender, String location, double charge, double alat,double along ) {
```

Comment:
In previous review, I suggested to change Guides class to use the builder pattern. What's your thought on that? https://github.com/abdullahalrifat/team-37-codeu/pull/17